core/numbers/numbers.py

```diff
@@ -177,7 +177,7 @@ def split_list(value, l: list) -> Iterator:
 number_small_map = {n: i for i, n in enumerate(number_small_list)}
 
 mod.list("number_small", desc="List of small numbers")
-mod.tag("prefixed_numbers", desc="Require prefix when saying a number")
+mod.tag("unprefixed_numbers", desc="Dont require prefix when saying a number")
```

Comment:
Minor nitpick: should numbers.talon be disabled when this tag is active?

Reply:
I'm not sure. I don't know if people want both active or only one of them?